eml/components/login/LoginForm.js

```diff
@@ -33,28 +30,6 @@ export default function LoginForm() {
   // State variable to track password visibility
   const [showPassword, setShowPassword] = useState(false);
 
-  /**
-   * Stores the user info in async storage
-   * @param {*} userInfo: {id, firstName, lastName, email}
-   */
-  async function saveUserInfoLocally(userInfo) {
```

Comment:
Is it safe to delete this? We still need to save some user info locally?

Reply:
the deleted function has the same functionality as setUserInfo (line 56), which has been imported from StorageServices